fastly/kv_store.go

```diff
@@ -388,6 +395,67 @@ func (c *Client) GetKVStoreKey(i *GetKVStoreKeyInput) (string, error) {
 	return string(output), nil
 }
 
+// GetKVStoreItemInput is the input to the GetKVStoreItem function.
+type GetKVStoreItemInput struct {
+	// StoreID is the StoreID of the kv store (required).
+	StoreID string
+	// Key is the key of the item to fetch (required).
+	Key string
+}
+
+// GetKVStoreItemOutput is the output of the GetKVStoreItem function.
+type GetKVStoreItemOutput struct {
+	// Value is the value stored in the item.
+	Value io.ReadCloser
```

Comment:
I would add that this needs to be closed (or consumed via the `ValueAs...` methods) to avoid leaking resources.